### ISSUE TITLE
some code files in lib don't have copyright headers

### ISSUE THREAD

**Issue by fduncanh** [FD-/RPiPlay#350]
maybe nitpicking, but it would be good to fix this

* EDIT: these are all from juvoh code.   I will fix these with juvoh LGPL header in the uxplay source taken from rpiplay, and add a PR here.

(juvoh committed aug 8, 2019 without copyright header     includes playfair.h which is GPLv3  does this make these
lib/fairplay-playfair.c
lib/fairplay.h

suggestion these should be LGPL with juvoh header , as they could in principle link to an altenate LGPL version of plairfair.h in the event that such a thing became available.   All code in lib except for that in  subdirectory fairplay should be LGPL.

---------------------------------------------
(juvoh)    
from   shariplay/include/shairplay,  added  oct 5, 2013 without copyright header.  assume needs juvoh LGPL header
raop.h
global.h
dnssd.h   added Mar 9 2012

-------------------
(juvoh) added Mar 9 2012, as a list of defined without copyright header.    does this need a header (if so LGPL)?

dnssdint.h

**Comment by FD-** [comment]
It's been a while since I started the project, so I don't remember specifics. However, from some quick research, it seems like all of these files except dnssd.h stem from juvoh's shairplay, but don't bear a copyright notice there either.
dnssd.h was created by me a few years ago IIRC, but only as part of restructuring. I filled it with code from other parts (don't remember details; must've been either dsafa22's or juvoh's code).

**Comment by fduncanh** [comment]
I'll do a PR here and fix in uxplay.

**Comment by fduncanh** [comment]
The first part of dnssd.h is in shairplay/include , so that is also its origin (part of juvoh initial commit)

**Comment by FD-** [comment]
I'm not sure about the legal perspective of someone other than the original author adding a copyright notice. If you want to clarify the source and license of the code, maybe just add something along the lines of "This code originates from the shairplay project by juhovh and contributors. It is used under permission of the ? license."

**Comment by fduncanh** [comment]
Hmm.   OK.    Since you added it to RPiPlay,  you have some authorship, at least for changes.
How about

(EDITED)
```                                                                                                                                                                           
/**
 *  This code originates from the Shairplay project
 *  https://github.con/juvoh/shairplay by Juho Vähä-Herttua and
 *  contributors (2011-2019), and is used under permission of that project's
 *  GNU Lesser General Public License (as described below) in the project:
 *
 *  RPiPlay - An open-source AirPlay mirroring server for Raspberry Pi
 *  Modifications (C) Florian Draschbacher and contributors (2019-2021)
 *
 *  This library is free software; you can redistribute it and/or
 *  modify it under the terms of the GNU Lesser General Public
 *  License as published by the Free Software Foundation; either
 *  version 2.1 of the License, or (at your option) any later version.
 *
 *  This library is distributed in the hope that it will be useful,
 *  but WITHOUT ANY WARRANTY; without even the implied warranty of
 *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
 *  Lesser General Public License for more details.
 */

```

I'll do a PR for this on rpiplay, and then use on uxplay ,

UxPlay is now in debian and ubuntu, and working great on 32/64 bit  R Pi (at least models 4B and 3) with the fixes to gstreamer's Video4Linux2 plugin, some of which I pushed into the development branch, and have now been backported to the gstreamer-1.18.4 updates in Raspberry Pi OS (Buster).  (backports are available as patches for 1.20.x (needed by other distributions like Ubuntu and Manjaro)at the UxPlay site)

**Comment by fduncanh** [comment]
submitted as PR #351
